src/transform/FooterReadMore.css

```diff
@@ -0,0 +1,92 @@
+
+.pagelib_footer_readmore_page {
+  background-color: white;
+  padding: 10px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  padding-top: 11px;
+  padding-bottom: 11px;
+  border-radius: 2px;
+  box-shadow: 0px 2px 4px rgba(200, 204, 209, 0.5);
+  display:block;
+}
+
+@media (min-device-width:768px){
+  .pagelib_footer_readmore_page {
+    box-shadow: 0 0 6px rgba(200, 204, 209, 0.5);
+  }
+}
+
+.pagelib_footer_readmore_page_container {
+
+}
+
+.pagelib_footer_readmore_page_title {
+  color: #222222;
+  display: block;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.pagelib_footer_readmore_page_description {
+  color: #a2a9b1;
+  display: block;
+  font-size: 0.9em;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+}
+
+.pagelib_footer_readmore_page_description::first-letter {
+  text-transform: uppercase;
+}
+
+.pagelib_footer_readmore_bookmark_filled {
+    background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAABGCAYAAACKeRdbAAAABGdBTUEAALGPC/xhBQAAAAFzUkdCAK7OHOkAAAAgY0hSTQAAeiYAAICEAAD6AAAAgOgAAHUwAADqYAAAOpgAABdwnLpRPAAAAAZiS0dEAP8A/wD/oL2nkwAAAqhJREFUaN7t20lrE3EYx/HfzGQyM5nsNmkTEo2JJmmSCW5RRHsIir20B0GKCuJFg6CHllKV1rbULBeh4EFcMCIEN2xLjEevHn0NBREEwfcQDzWFUJNMNp8E/s8tQ5L/9xMmsxyGO575VsUQD08dwADUAQxAHcAA1AEMQB3AANQBDEAdwADUAQxAHcAA1AEMQB3AANQBDEAdwADUAQxAHcAA1AEMQB3AANQBDEAdwADUAQxAHcAA1AEMQB0w0IDJk05cSDn7CjD064tn0m7cvewHAJgkHuWvv4cHkJn2IjPl2X394NoBqIqAN19+DT5gfsaPK+fce7bPXfLBLAt4/vnngAI4YPX6QUyfbrzP35zyQJUFrH/8MXiAR7dCSB+xt3zf1fNuKDKPfOl7T9bt+ijEccCT2bCu+NpcPDuC/I0gOGoAzwHFhShOjVva/uxkyoH126GdX4ACIPBAaSmGZEjtePGJpB3PZg93ZegIYDTw+LAaR8SvdL7y3zkRtaC4EAXfIaJtgGzksZlNIDAmdx1fm2RIRWkpBoPQvqItgCoLKOc0eJxiz+JrE/EreL8Sg2hoD6EbYDUJqBQ0jNj6dvWBwJiMzYcJSKJ+hC6A0yqiUtBgU4W+xdfGu8+Ick6DSda3VkvAqEPEp1wCZqX/8bVx2UVU8glYTa3XbArwuSRsZTUo0v+/bbCbDagUNDgszf9vDcuCHhkba/G29sdej1kRUMknMOpojPgnILLfhHcr8Y4Oa70eReKxldXgc0n6AMmgitLiOIQButmURA4ba3GEvHtPnHWZqagFr+51flbs5xgEDm+XY4gH6i9ddgFnNBuezoWpO5uOwAOv70dxLGyuB6SPOvD4ziHqPl3DccCL+QgmkvadDcvF7eqwzuLL7SpXrVaH+iGgP5QPEShTwCAqAAAAAElFTkSuQmCC);
```

Comment:
I checked and it doesn't look like any of the menu icons need to be mirrored when RTL.